packages/jsii/test/negatives/neg.implements-class.ts

```diff
@@ -1,4 +1,4 @@
-///!MATCH_ERROR: Implements clause of jsii.TryingToImplementClass uses jsii.NotAnInterface as an interface
+///!MATCH_ERROR: Inheritence clause of jsii.TryingToImplementClass uses jsii.NotAnInterface as an interface
```

Comment:
I reckon this should be `Inheritance` 😞 (Obv. this comes from the error message :D)

Reply:
fixed